cypress/integration/cbl/tasks_manager.js

```diff
@@ -7,8 +7,8 @@ describe('CBL: Tasks Manager Test', () => {
 
     // authenticate as 'teacher' user
     beforeEach(() => {
-        cy.server().route('GET', '/cbl/tasks').as('taskData');
-        cy.server().route('POST', '/cbl/tasks/save*').as('taskSave');
+        cy.intercept('GET', '/cbl/tasks?!(\\?*)').as('taskData');
+        cy.intercept('POST', '/cbl/tasks/save?!(\\?*)').as('taskSave');
```

Comment:
@Camus1859 `?!(\\?*)` doesn't seem to be doing anything—`?!` is not a documented prefix with any meaning I can find here: https://github.com/motemen/minimatch-cheat-sheet

So I think this pattern is saying:
- There isn't `?*` at the end
- The above may or may not be true

Looking at actual calls being made during this test, these requests do have misc `?query=params` appended that we want to allow for but not make a hard requirement, so the pattern I showed in #671 is more appropriate